_episodes_rmd/12-supp-factors.Rmd

```diff
@@ -193,7 +193,7 @@ dat$Gender[dat$Gender == 'M'] <- 'm'
 
 > ## Updating Factors
 >
-> ```{r updating-factors}
+> ```{r updating-factors, fig. alt = "Barplot graph showing blood pressure in males and females.}
```

Comment:
```suggestion
> ```{r updating-factors, fig.alt = "Barplot graph showing blood pressure in males and females.}
```